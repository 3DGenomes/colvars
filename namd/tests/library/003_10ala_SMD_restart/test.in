colvarsTrajFrequency    1
#colvarsRestartFrequency 10


colvar {
    name rmsd

    outputSystemForce on
    outputAppliedForce on
    width 0.01

    lowerBoundary 0.0
    upperBoundary 0.1

    hardLowerBoundary yes
    expandBoundaries yes

#    lowerWallConstant 10.0
    upperWallConstant 10.0

    rmsd {
        atoms {
            enablefitGradients yes
            psfSegID BH
            atomNameResidueRange CA 1-10
        }
        refpositionsfile ../Common/pi-ideal.pdb
    }
} 

harmonic {
    colvars        rmsd
    centers        0.0
    targetCenters  0.2
    targetNumSteps 20
    outputEnergy   yes
    outputAccumulatedWork  yes
    outputCenters  yes
}
