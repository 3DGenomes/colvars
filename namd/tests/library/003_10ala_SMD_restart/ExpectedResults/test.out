Charm++: standalone mode (not using charmrun)
Converse/Charm++ Commit ID: v6.4.0-beta1-0-g5776d21
CharmLB> Load balancer assumes all CPUs are same.
Charm++> Running on 1 unique compute nodes (6-way SMP).
Charm++> cpu topology info is gathered in 0.000 seconds.
Info: NAMD 2.9 for Linux-x86_64-multicore
Info: 
Info: Please visit http://www.ks.uiuc.edu/Research/namd/
Info: for updates, documentation, and support information.
Info: 
Info: Please cite Phillips et al., J. Comp. Chem. 26:1781-1802 (2005)
Info: in all publications reporting results obtained with NAMD.
Info: 
Info: Based on Charm++/Converse 60400 for multicore-linux64
Info: Built Thu Jun 13 17:22:49 EDT 2013 by gfiorin on helicon
Info: 1 NAMD  2.9  Linux-x86_64-multicore  1    helicon  gfiorin
Info: Running on 1 processors, 1 nodes, 1 physical nodes.
Info: CPU topology information available.
Info: Charm++/Converse parallel runtime startup completed at 0.00151801 s
Info: 40.8164 MB of memory in use based on /proc/self/stat
Info: Configuration file is test.namd
Info: Working in the current directory /home/gfiorin/sources/colvars/namd/tests/library/003_10ala_SMD_restart
TCL: Suspending until startup complete.
Info: SIMULATION PARAMETERS:
Info: TIMESTEP               0.5
Info: NUMBER OF STEPS        10
Info: STEPS PER CYCLE        20
Info: LOAD BALANCER  Centralized
Info: LOAD BALANCING STRATEGY  New Load Balancers -- DEFAULT
Info: LDB PERIOD             4000 steps
Info: FIRST LDB TIMESTEP     100
Info: LAST LDB TIMESTEP     -1
Info: LDB BACKGROUND SCALING 1
Info: HOM BACKGROUND SCALING 1
Info: MIN ATOMS PER PATCH    40
Info: INITIAL TEMPERATURE    250
Info: CENTER OF MASS MOVING INITIALLY? NO
Info: DIELECTRIC             1
Info: EXCLUDE                SCALED ONE-FOUR
Info: 1-4 ELECTROSTATICS SCALED BY 1
Info: MODIFIED 1-4 VDW PARAMETERS WILL BE USED
Info: DCD FILENAME           test.dcd
Info: DCD FREQUENCY          2000
Info: DCD FIRST STEP         2000
Info: NO EXTENDED SYSTEM TRAJECTORY OUTPUT
Info: NO VELOCITY DCD OUTPUT
Info: NO FORCE DCD OUTPUT
Info: OUTPUT FILENAME        test
Info: BINARY OUTPUT FILES WILL BE USED
Info: RESTART FILENAME       test
Info: RESTART FREQUENCY      40000
Info: BINARY RESTART FILES WILL BE USED
Info: SWITCHING ACTIVE
Info: SWITCHING ON           9
Info: SWITCHING OFF          11
Info: PAIRLIST DISTANCE      12
Info: PAIRLIST SHRINK RATE   0.01
Info: PAIRLIST GROW RATE     0.01
Info: PAIRLIST TRIGGER       0.3
Info: PAIRLISTS PER CYCLE    2
Info: PAIRLISTS ENABLED
Info: MARGIN                 1
Info: HYDROGEN GROUP CUTOFF  2.8
Info: PATCH DIMENSION        15.8
Info: CROSSTERM ENERGY INCLUDED IN DIHEDRAL
Info: TIMING OUTPUT STEPS    1
Info: COLLECTIVE VARIABLES CALCULATION REQUESTED
Info: COLLECTIVE VARIABLES CONFIGURATION   test.in
Info: LANGEVIN DYNAMICS ACTIVE
Info: LANGEVIN TEMPERATURE   300
Info: LANGEVIN DAMPING COEFFICIENT IS 10 INVERSE PS
Info: LANGEVIN DYNAMICS APPLIED TO HYDROGENS
Info: USING VERLET I (r-RESPA) MTS SCHEME.
Info: C1 SPLITTING OF LONG RANGE ELECTROSTATICS
Info: PLACING ATOMS IN PATCHES BY HYDROGEN GROUPS
Info: RANDOM NUMBER SEED     87654321
Info: USE HYDROGEN BONDS?    NO
Info: COORDINATE PDB         ../Common/da.pdb
Info: STRUCTURE FILE         ../Common/da.psf
Info: PARAMETER file: CHARMM format! 
Info: PARAMETERS             ../Common/par_all22_prot.inp
Info: USING ARITHMETIC MEAN TO COMBINE L-J SIGMA PARAMETERS
Info: SUMMARY OF PARAMETERS:
Info: 140 BONDS
Info: 356 ANGLES
Info: 459 DIHEDRAL
Info: 43 IMPROPER
Info: 6 CROSSTERM
Info: 95 VDW
Info: 0 VDW_PAIRS
Info: 0 NBTHOLE_PAIRS
Info: TIME FOR READING PSF FILE: 0.00156784
Info: TIME FOR READING PDB FILE: 0.000158787
Info: 
Info: ****************************
Info: STRUCTURE SUMMARY:
Info: 104 ATOMS
Info: 103 BONDS
Info: 183 ANGLES
Info: 251 DIHEDRALS
Info: 22 IMPROPERS
Info: 0 CROSSTERMS
Info: 0 EXCLUSIONS
Info: 312 DEGREES OF FREEDOM
Info: 51 HYDROGEN GROUPS
Info: 4 ATOMS IN LARGEST HYDROGEN GROUP
Info: 51 MIGRATION GROUPS
Info: 4 ATOMS IN LARGEST MIGRATION GROUP
Info: TOTAL MASS = 727.821 amu
Info: TOTAL CHARGE = 6.70552e-08 e
Info: *****************************
Info: 
Info: Entering startup at 0.015409 s, 56.4141 MB of memory in use
Info: Startup phase 0 took 4.00543e-05 s, 56.4141 MB of memory in use
Info: Startup phase 1 took 0.000168085 s, 56.4141 MB of memory in use
Info: Startup phase 2 took 4.98295e-05 s, 56.4141 MB of memory in use
Info: Startup phase 3 took 3.21865e-05 s, 56.4141 MB of memory in use
Info: PATCH GRID IS 2 BY 1 BY 1
Info: PATCH GRID IS 1-AWAY BY 1-AWAY BY 1-AWAY
Info: REMOVING COM VELOCITY 0.177776 1.11835 0.497162
Info: LARGEST PATCH (1) HAS 54 ATOMS
Info: Startup phase 4 took 0.000134945 s, 56.5781 MB of memory in use
Info: Startup phase 5 took 3.50475e-05 s, 56.5781 MB of memory in use
Info: Startup phase 6 took 2.88486e-05 s, 56.5781 MB of memory in use
LDB: Central LB being created...
Info: Startup phase 7 took 3.91006e-05 s, 56.5781 MB of memory in use
Info: CREATING 19 COMPUTE OBJECTS
Info: NONBONDED TABLE R-SQUARED SPACING: 0.0625
Info: NONBONDED TABLE SIZE: 705 POINTS
Info: ABSOLUTE IMPRECISION IN VDWB TABLE ENERGY: 1.03398e-25 AT 10.9972
Info: RELATIVE IMPRECISION IN VDWB TABLE ENERGY: 1.12007e-16 AT 10.9972
colvars: ----------------------------------------------------------------------
colvars: Initializing the collective variables module, version 2013-06-13.
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsd
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:   All components initialized.
colvars:   # width = 0.01
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0.1
colvars:   # upperWallConstant = 10
colvars:   # upperWall = 0.1 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd }
colvars:   # outputEnergy = on
colvars:   # forceConstant = 1 [default]
colvars:   The force constant for colvar "rmsd" will be rescaled to 10000 according to the specified width.
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0.2 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = on
colvars:   # outputAccumulatedWork = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module initialized.
colvars: ----------------------------------------------------------------------
Info: Startup phase 8 took 0.00285387 s, 57.1797 MB of memory in use
Info: Startup phase 9 took 4.1008e-05 s, 57.1797 MB of memory in use
Info: Startup phase 10 took 4.1008e-05 s, 57.5156 MB of memory in use
Info: Finished startup at 0.018873 s, 57.5156 MB of memory in use

ETITLE:      TS           BOND          ANGLE          DIHED          IMPRP               ELECT            VDW       BOUNDARY           MISC        KINETIC               TOTAL           TEMP      POTENTIAL         TOTAL3        TEMPAVG

ENERGY:       0        17.7452        62.9761        30.7071         8.3800             62.2194       -13.6978         0.0000         0.0000        69.9033            238.2333       225.4931       168.3300       238.2948       225.4931

TIMING: 1  CPU: 0, 0/step  Wall: 0.0011611, 0.000343084/step, 8.57711e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       1        18.4340        61.7524        30.6385         8.0406             62.2052       -13.7499         0.0000         0.2337        71.5741            239.1286       230.8830       167.5545       239.4270       230.8830

TIMING: 2  CPU: 0, 0/step  Wall: 0.00149989, 0.000338793/step, 7.52873e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       2        20.7869        60.4346        30.5404         7.6435             62.1552       -13.7995         0.0000         0.9206        72.2735            240.9553       233.1389       168.6818       241.2924       233.1389

TIMING: 3  CPU: 0, 0/step  Wall: 0.00183105, 0.000331163/step, 6.43929e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       3        24.0593        59.0514        30.4316         7.2257             62.0598       -13.8541         0.0000         2.0410        72.1402            243.1549       232.7088       171.0148       243.2727       232.7088

TIMING: 4  CPU: 0, 0/step  Wall: 0.00216198, 0.000330925/step, 5.51542e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       4        27.0608        57.7542        30.3163         6.7965             61.9990       -13.8952         0.0000         3.5805        74.2325            247.8445       239.4583       173.6120       248.6783       239.4583

TIMING: 5  CPU: 0, 0/step  Wall: 0.00249004, 0.000328064/step, 4.55644e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       5        28.8574        56.4792        30.1925         6.3302             61.9221       -13.9644         0.0000         5.5269        77.2610            252.6049       249.2276       175.3439       253.1238       249.2276

TIMING: 6  CPU: 0, 0/step  Wall: 0.00281906, 0.000329018/step, 3.65575e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       6        29.1804        55.3328        30.0673         5.8630             61.8501       -14.0581         0.0000         7.7855        81.6203            257.6414       263.2899       176.0211       257.9895       263.2899

TIMING: 7  CPU: 0, 0/step  Wall: 0.00314689, 0.000327826/step, 2.73188e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       7        28.1881        54.4303        29.9634         5.4446             61.7739       -14.1402         0.0000        10.2843        83.7908            259.7352       270.2912       175.9444       258.5400       270.2912

TIMING: 8  CPU: 0, 0/step  Wall: 0.003474, 0.00032711/step, 1.81728e-07 hours remaining, 57.644531 MB of memory in use.
ENERGY:       8        26.6711        53.9581        29.8601         5.0596             61.6934       -14.2228         0.0000        13.0618        88.1841            264.2653       284.4632       176.0812       264.0482       284.4632

TIMING: 9  CPU: 0, 0/step  Wall: 0.00380301, 0.000329018/step, 9.13938e-08 hours remaining, 57.644531 MB of memory in use.
ENERGY:       9        25.4450        53.9573        29.7862         4.7374             61.5978       -14.3063         0.0000        15.9884        90.7093            267.9151       292.6091       177.2057       267.1492       292.6091

colvars: Saving collective variables state to "test.colvars.state".
TIMING: 10  CPU: 0, 0/step  Wall: 0.00421786, 0.000414848/step, 0 hours remaining, 57.644531 MB of memory in use.
ETITLE:      TS           BOND          ANGLE          DIHED          IMPRP               ELECT            VDW       BOUNDARY           MISC        KINETIC               TOTAL           TEMP      POTENTIAL         TOTAL3        TEMPAVG

ENERGY:      10        25.4997        54.3850        29.7396         4.4824             61.5066       -14.4060         0.0000        19.0566        92.9944            273.2583       299.9802       180.2639       273.0810       299.9802

WRITING EXTENDED SYSTEM TO OUTPUT FILE AT STEP 10
WRITING COORDINATES TO OUTPUT FILE AT STEP 10
COORDINATE DCD FILE WAS NOT CREATED
The last position output (seq=-2) takes 0.000 seconds, 57.645 MB of memory in use
WRITING VELOCITIES TO OUTPUT FILE AT STEP 10
The last velocity output (seq=-2) takes 0.000 seconds, 57.645 MB of memory in use
====================================================

WallClock: 0.022869  CPUTime: 0.024001  Memory: 57.644531 MB
Program finished.
